contributors/devel/welcome-to-kubernetes-new-developer-guide.md

```diff
@@ -0,0 +1,126 @@
+
+Welcome to Kubernetes! (New Developer Guide)
+============================================
+
+2017-06-21 through 2017-08-06
+Original author: [Alexander Campbell](https://github.com/alexandercampbell)
```

Comment:
My justification for putting my name on things: I'm not a fan of the way the docs end up floating around ownerless and unmaintained.

I can remove the dates for sure. What do you think?